lib/veritech-client/src/lib.rs

```diff
@@ -212,31 +214,69 @@ impl Client {
         .await
     }
 
-    async fn execute_request<R>(
+    async fn execute_request<R: Serialize + CycloneRequestable<Response: DeserializeOwned>>(
         &self,
         subject: Subject,
         output_tx: Option<mpsc::Sender<OutputStream>>,
         request: &R,
         request_mode: RequestMode,
-    ) -> ClientResult<FunctionResult<R::Response>>
-    where
-        R: Serialize + CycloneRequestable,
-        R::Response: DeserializeOwned,
-    {
+    ) -> ClientResult<FunctionResult<R::Response>> {
+        // Subscribe to responses and send the request. These unsubscribe when dropped.
+        let (root_subscriber, mut result_subscriber, output_subscriber) =
+            self.send_request(subject, request, request_mode).await?;
+
+        // Forward output messages in the background
+        let cancel_output = CancellationToken::new();
+        if let Some(output_tx) = output_tx {
+            let cancel_output = cancel_output.clone();
+            tokio::spawn(cancel_output.run_until_cancelled_owned(forward_output(
+                output_subscriber,
+                output_tx,
+                Span::current(),
+            )));
```

Comment:
Eventually we should use tokens and trackers for spawned tasks, but given that `veritech-client` already had `tokio::spawn` usage and both the dal and sdf don't have the best mechanisms for spawned task management, this should be fine for now!